lib/savon/operation.rb

```diff
@@ -101,9 +101,10 @@ def build_request(builder)
       request.body = builder.to_s
 
       if builder.multipart
-        request.gzip
+        type = @locals[:mtom] ? 'application/xop+xml"; start-info="text/xml' : SOAP_REQUEST_TYPE[@globals[:soap_version]]
```

Comment:
Minor: this MTOM mime type could also be extracted to a constant, so that both branches are harmonious, looking uniform.